p5/core/tests/test_geometry.py

```diff
@@ -54,10 +58,10 @@ def test_vertices(self):
                                                     0.5, -0.5], [-0.5, 0.5, 0.5],
             [0.5, -0.5, -0.5], [0.5, 0.5, -0.5], [0.5, -0.5, 0.5], [0.5, 0.5, 0.5],
             [-0.5, -0.5, -0.5], [0.5, -0.5, -
-                                 0.5], [-0.5, -0.5, 0.5], [0.5, -0.5, 0.5],
+            0.5], [-0.5, -0.5, 0.5], [0.5, -0.5, 0.5],
```

Comment:
could you add comments for which vertex represents which corner of a cube ? Could improve code readability.

Reply:
Yes sure, a lot of code is not documented in p5py, but I will add the comments after my internals :/